docs/docs/manual/facets.md

```diff
@@ -0,0 +1,284 @@
+
+OpenRefine has several default facets, which you’ll learn about below. The most powerful facets are the ones designed by you - custom facets, written using [expressions](expressions) to transform the data behind the scenes and help you narrow down to precisely what you’re looking for. 
+
+Facets are not saved in the project along with the data. But you can save a *permalink* to the current state of the application. Find the "permalink" next to the project’s name.
+
+You can modify any facet expression by clicking the “change” button to the right of the column name in the facet sidebar.
+
+Facet boxes that appear in the sidebar can be resized and rearranged. You can drag and drop the title bar of each box to reorder them, and drag on the bottom bar of text facet boxes. 
+
+## Text facet
+
+A text facet can be generated on any column with the “text” data type. Select the column dropdown and go to “Facet” > “Text facet”. The created facet will be sorted alphabetically, and can be sorted by count. 
+
+A text facet is very simple: it takes the total contents of the cells of the column in question and matches them up. It does no guessing about typos or near-matches. 
+
+You can edit any entry that appears in the facet display, by hovering over the facet and clicking the “edit” button that appears. This will mass-edit every matching cell in your data. This is a great way to fix typos, whitespace, and other issues that may be affecting the way facets appear. You can also automate the cleanup of facets by using [clustering](transforming#cluster-and-edit), with the “Cluster” button displayed within the facet window. 
+
+Each text facet shows up to 2,000 choices by default. You can [increase this limit on the Preferences screen](running#preferences) if you need to, which will increase the processing work required by your browser. If your applied facet has more choices than the current limit, you'll be offered the option to increase the limit, which will edit that preference for you. 
+
+The choices and counts displayed in each facet can be copied as tab-separated values. To do so, click on the "X choices" link near the top left corner of the facet.
+
+## Numeric facet
+
+![A screenshot of an example numeric facet.](/img/numericfacet.png)
+
+Whereas a text facet groups unique text values into groups, a numeric facet sorts numbers by their range - smallest to biggest. This displays visually and allows you to set a custom facet within that range. You can drag the minimum and maximum range markers to set a range. OpenRefine snaps to some basic equal-sized divisions - 19 in the example set above. 
+
+You will be offered the option to include blank, non-numeric, and error values in your numeric visualization; these will appear in the visual range as “0” values.
```

Comment:
I don't think this is correct. Blank, non-numeric and error values don't appear as zero values - they aren't included in the bar chart - they are included/excluded based on checking the relevant boxes